pkg/reconciler/tenancy/initialization/apis/initialization/v1alpha1/types.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/apis/conditions/v1alpha1"
+)
+
+// APISet records a set of API Bindings that need to be created
+// and bound before initialization is complete. This configuration
+// applies to all workspaces of the type that shares a name with this
+// object.
+//
+// +crd
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:resource:scope=Cluster,categories=kcp
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+type APISet struct {
```

Comment:
Bikeshed begins :)

Reply:
`InitialAPIBinding`...?

Reply:
ClusterWorkspaceTypeAPIBindings

Reply:
Not a fan of types that end-up with an `s`.
What about `ClusterWorkspaceTypeAPIBindingSet` ?